packages/zowe-explorer/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ## TBD Release
 
+- Added Job search label to the Jobs tree.
```

Comment:
Should this be under the new enhancements section?
Or maybe we just update the related entry to also mention the label updates?

Reply:
I moved this to new enhancements section